kpagacz/wiki-on-fire
# add: backend endpoint for the most viewed articles and the http layer

Closes #179


How to test:
* make sure to undo the seeds to the database and seed it again with the new and updated seeds. In the console:
```
npx sequelize db:seed:undo:all
npx sequelize db:migrate:undo:all
npm run migrate
npm run seed
```
* you can easily test the backend function on the development environment using this:
```js
import { getMostViewed } from "./services/articleService.js";

const res = await getMostViewed(2, 10);
console.dir(res);
```
* testing the function from the frontend is a little bit more tricky. I usually run the backend, add "type" : "module" entry to the frontend/package.json and run a test file:
```js
import { getMostViewedArticles } from "./mostViewedArticles.http.js";

const res = await getMostViewedArticles(1, 10);
console.dir(res);
```
